packages/web-components/src/drawer/drawer.ts

```diff
@@ -8,76 +8,66 @@ import { DrawerPosition, DrawerSize, DrawerType } from './drawer.options.js';
  *
  * @extends FASTElement
  *
- * @attr {DrawerType} type - Determines whether the drawer should be displayed as modal, non-modal, or alert.
- * @attr {DrawerPosition} position - Sets the position of the drawer (start/end).
- * @attr {DrawerSize} size - Sets the size of the drawer (small/medium/large).
- * @attr {string} ariaDescribedby - The ID of the element that describes the drawer.
- * @attr {string} ariaLabelledby - The ID of the element that labels the drawer.
- *
  * @csspart dialog - The dialog element of the drawer.
  *
  * @slot - Default slot for the content of the drawer.
  *
  * @fires toggle - Event emitted after the dialog's open state changes.
  * @fires beforetoggle - Event emitted before the dialog's open state changes.
  *
- * @method show - Method to show the drawer.
- * @method hide - Method to hide the drawer.
- * @method clickHandler - Handles click events on the drawer.
- * @method emitToggle - Emits an event after the dialog's open state changes.
- * @method emitBeforeToggle - Emits an event before the dialog's open state changes.
- *
  * @summary A component that provides a drawer for displaying content in a side panel.
  *
  * @tag fluent-drawer
  */
 export class Drawer extends FASTElement {
   /**
-   * @public
    * Determines whether the drawer should be displayed as modal or non-modal
    * When rendered as a modal, an overlay is applied over the rest of the view.
+   * @public
    */
   @attr
   public type: DrawerType = DrawerType.modal;
 
   /**
-   * @public
    * The ID of the element that labels the drawer.
+   * @public
    */
   @attr({ attribute: 'aria-labelledby' })
   public ariaLabelledby?: string;
 
   /**
-   * @public
    * The ID of the element that describes the drawer.
+   * @public
    */
   @attr({ attribute: 'aria-describedby' })
   public ariaDescribedby?: string;
 
   /**""
+   * Sets the position of the drawer (start/end).
    * @public
    * @defaultValue start
-   * Sets the position of the drawer (start/end).
    */
   @attr
   public position: DrawerPosition = DrawerPosition.start;
 
   /**
+   * Sets the size of the drawer (small/medium/large).
    * @public
    * @defaultValue medium
```

Comment:
nit: the `@defaultValue` can be removed since it's indicated by the property assignment.